### ISSUE TITLE
[Feature Request] Make the "Map Packs" list have checkboxes like the Mods list does

### ISSUE THREAD

**Issue by msmigiel** [Youda008/DoomRunner#156]
Please consider making the map packs list selectable by checkboxes, similar to how the mods list works. To me, it is a little more intuitive to use checkboxes and it allows quicker, one handed selections and deselections. It also took me a minute to figure out how to not select any map pack (must hold control and click the last one) but with a checkbox, it would be a little more intuitive.

**Comment by thecnoNSMB** [comment]
Seconded, deselecting a map/map pack is a little awkward at present. It probably should be an option, though, since having checkboxes might make it more likely you accidentally select multiple incompatible map packs.

**Comment by Youda008** [comment]
This is kinda double-edged. While it's more intuitive, when you need to deselect the map file, it's also more clicking everytime you want to switch one map pack for another, because you need to untick the previous one first, and it's also likely that you will forget.
But i will think about the possibility of making it optional.

**Comment by Youda008** [comment]
I'm sorry, but i don't have the energy to do this anymore.
I will leave this open if some other volunteer takes over, but it will not be me who does this.

**Comment by aulMasa** [comment]
The first item of the list can by a <no map> item. And it will deselect in one click without changing the current behaviour.

**Comment by yum13241** [comment]
Just drag it to the mod list pane, that's what I do, as it saves me from having to open a file picker dialog.

**Comment by thecnoNSMB** [comment]
> Just drag it to the mod list pane, that's what I do, as it saves me from having to open a file picker dialog.

That would work, but it's not practical if you have a significant amount of map WADs.

**Comment by thecnoNSMB** [comment]
By "significant amount" I meant, like, thousands.

**Comment by aulMasa** [comment]
Drag the maps you want to play, not everything

**Comment by yum13241** [comment]
yeah lol, plus even the file picker dialog would force you to do the same.

OOOOOOORRRR you're loading too much for your own good.

**Comment by thecnoNSMB** [comment]
Ah, I see what you mean. I thought you were suggesting to load all available maps into the mod list and use its checkboxes to select the ones you want.
